### DISCUSSION TITLE
"Invalid or corruped jarfile" when trying to run the Fabric jar

### DISCUSSION THREAD

**Discussion by beanniee** [FabricMC/fabric-api#1602] [Player support]
I was trying to run Fabric 1.16.1 via the minecraft launcher, but it would always crash. I tried to check the Fabric file, and when I double clicked the .jar file i got this error:
![Capture](https://user-images.githubusercontent.com/43115655/127523101-e2f769d7-4c6e-4f4f-ab9c-e8fca1a6080e.PNG)

**Comment by warjort** [comment]
Since you are running windows, try doing the following on the command line:
````
certutil -hashfile intermediary-fabric-loader-0.11.6-1.16.1.jar MD5
````
It should print a string that matches the contents of fabric-loader-0.11.6.jar.md5 here:
https://maven.fabricmc.net/net/fabricmc/fabric-loader/0.11.6/

If it doesn't match, it means the file didn't download correctly. Try reinstalling.
